src/tools/annotation/LengthTool.js

```diff
@@ -207,6 +199,8 @@ export default class LengthTool extends BaseAnnotationTool {
         // So that it sits beside the length tool handle
         const xOffset = 10;
 
+        this.updateStatistics(evt, data);
```

Comment:
If we're going to continue calling this in the render loop, should we have a dirty "invalidated" flag we can use to prevent unnecessary recalcs?